include/secp256k1.h

```diff
@@ -268,7 +268,7 @@ SECP256K1_API void secp256k1_context_destroy(
  *  In:   fun:  a pointer to a function to call when an illegal argument is
  *              passed to the API, taking a message and an opaque pointer.
  *              (NULL restores the default handler.)
- *        data: the opaque pointer to pass to fun above.
+ *        data: the opaque pointer to pass to fun above (can be NULL for default handler)
```

Comment:
I think we could say "must be NULL" for default handler because the default handler does not expect data. (Again, nothing bad happens if you don't obey.)